src/main/java/jpa/domain/Line.java

```diff
@@ -25,14 +30,21 @@
 
     private String color;
 
-    public Line() {
+    @OneToMany(fetch = FetchType.LAZY, mappedBy = "line")
+    private List<LineStation> lineStations = new ArrayList<>();
+
+    protected Line() {
     }
 
     public Line(String name, String color) {
         this.name = name;
         this.color = color;
     }
 
+    public void addLineStation(LineStation lineStation) {
```

Comment:
```suggestion
    public void add(LineStation lineStation) {
```
파라미터 타입이 LineStation이니 메서드명에는 넣지 않아도 될 것 같습니다
그리고 add 보다 도메인 용어로 메서드명을 지어보면 어떨까요?

Reply:
그리고 lineStation가 null인지 체크하는 방어 로직도 추가해보면 좋을 것 같네요

Reply:
도메인 용어가 마땅히 생각나는게 없어서 수정을 못했는데 추천 해주실 수 있으신가요
add 말고는 마땅히 생각나는게 없네요 ㅠㅠ

Reply:
저는 도메인에 대한 지식이 없을 때 구글링으로 실제 용어를 어떻게 쓰는지 찾아보는 편입니다.
영문 위키피디아에서 Seoul Subway Line 2 같은 검색어로 찾아보니 몇 개 사용할만한 단어가 보이네요
제 의견과 달리 검색해도 괜찮은 용어가 없다면 add를 그대로 사용하셔도 됩니다.
제 의견이 중요한건 아니니 상배님이 판단하셔서 도메인 용어를 정하시면 됩니다